setup.py

```diff
@@ -211,7 +211,8 @@ def run(self):
             'torch': [
                 'torch>=1.7.0a0',
                 'torchvision>=0.8.0a0',
-                'transformers>=4.3.3, <4.23.0',
+                'tokenizers <= 0.20.3',
```

Comment:
move it before torch? the list is sorted alphabetically, and remove space around "<=" to keep consistent?